api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -96,10 +99,28 @@ message DownstreamTlsContext {
 }
 
 // TLS context shared by both client and server TLS contexts.
-// [#next-free-field: 9]
+// [#next-free-field: 11]
 message CommonTlsContext {
   option (udpa.annotations.versioning).previous_message_type = "envoy.api.v2.auth.CommonTlsContext";
 
+  // Config for CA client plugin to get secrets
+  message CAClientPluginConfig {
+    // plugin name
+    string name = 1;
+
+    // opaque name to identify a secret when workload needs more than one secret
+    string secret_name = 2;
+
+    // plugin specific config based on the plugin name
+    google.protobuf.Any typed_config = 3;
+  }
+
+  // the typed_config for a plugin with the name “GoogleManagedCA”
```

Comment:
Why is this in the OSS API? I think we can only accept this if there is some public implementation and reference material.

Reply:
Removing it. May consider adding it once we have a reference implementation.